packages/hop-node/src/db/TransferRootsDb.ts

```diff
@@ -484,16 +484,26 @@ class TransferRootsDb extends BaseDb {
         return false
       }
 
-      let timestampOk = true
+      // Since bonding of transferRoots is not time sensitive, wait an arbitrary amount of time for
+      // finality before attempting to bond. This prevents repetitive RPC calls, since that is the
+      // only true way to know finality for ORUs. The arbitrary time should represent roughly how long
+      // the longest chain should wait for finality.
+      let finalityTimestampOk = false
+      if (item?.committedAt) {
+        const longestTimeToFinalityMs = 2 * TenMinutesMs
```

Comment:
suggestion: instead of the hardcoded 10 minutes, maybe finality times can be set in config to make this value be computed (feel free to ignore this suggestion if it doesn't make sense to do so)

```js
const finalityMinutes = { ethereum: 1, optimism: 2, arbitrum: 3 }
const longestTimeToFinalityMs = (Math.max(...Object.values(finalityMinutes)) * 2) * 60 * 1000
````

Reply:
For these chains there is no consistency in finalized time since it depends on when in an epoch a tx is included. All these chains should (roughly) be the same since finalization of all of these relies on L1 finality. Based on that, and the fact that this is already arbitrary and just used in a non-sensitive way (to reduce RPC calls), I think it makes sense to just leave this arbitrary-ish value as a single constant here.